docs/configuration/trainer.md

```diff
@@ -19,18 +19,19 @@ By default, the ECD trainer is used.
         steps_per_checkpoint: 0
         regularization_lambda: 0
         regularization_type: l2
-        learning_rate: 0.001
-        reduce_learning_rate_on_plateau: 0
-        reduce_learning_rate_on_plateau_patience: 5
-        reduce_learning_rate_on_plateau_rate: 0.5
         increase_batch_size_on_plateau: 0
         increase_batch_size_on_plateau_patience: 5
         increase_batch_size_on_plateau_rate: 2
         increase_batch_size_on_plateau_max: 512
-        decay: false
-        decay_steps: 10000
-        decay_rate: 0.96
-        staircase: false
+        learning_rate: 0.001
+        learning_rate_scheduler:
+            decay: false
```

Comment:
`decay: false` is now `decay: null` in the new schema.

I would also say this config is misleading, as all the decay params do nothing. We might want to think about making it better reflect a realistic config.